android/src/main/java/com/it_nomads/fluttersecurestorage/FlutterSecureStoragePlugin.java

```diff
@@ -70,14 +71,14 @@ public void initInstance(BinaryMessenger messenger, Context context) {
     private void ensureInitStorageCipher() {
         if (storageCipher == null) {
             try {
-                Log.d(TAG, "Initializing StorageCipher");
+                if (BuildConfig.DEBUG) Log.d(TAG, "Initializing StorageCipher");
```

Comment:
Could we simply remove that?

Reply:
Sure, only this print or also the others?

Reply:
I think you should remove all prints altogether. Mundane tasks should not print anything. But instead there could be a switch or something(I don't know how hard it is), which user can use to enable/disable printing/logging.

Reply:
I'm still new to Flutter and Dart (and mobile and OOP). Are there different layers of logging; info, warn, error?

Reply:
@PieterAelse I would suggest to remove all prints. Agree with @drmirk. 

Ideally, the plugin should receive a parameter (probably, during initialization) with required Log level, which is off by default. Should not be hard, except to make sure it's consistent across all platforms.

Reply:
@mogol PR is updated, debug logs have been removed. Did not touch error logs (on crashes).